app/forms.py

```diff
@@ -0,0 +1,28 @@
+from flask_wtf import FlaskForm
+from wtforms import FloatField, TextField, MultipleFileField, TextAreaField
+from wtforms.validators import ValidationError
+
+
+class FormNewServices(FlaskForm):
+    name = TextField("name")
+    description = TextAreaField("description")
+    price = FloatField("price")
```

Comment:
I think the price field here should be in a different type? we need to have prices as an array of objects (item: price)